test/e2e/storage/testsuites/volumes.go

```diff
@@ -131,6 +136,10 @@ func (t *volumesTestSuite) defineTests(driver TestDriver, pattern testpatterns.T
 			l.testCleanup()
 			l.testCleanup = nil
 		}
+
+		// TODO(dyzz): At this point the volume still has not unmounted or detached...
```

Comment:
If the test cleanup checks that the Pod object is fully deleted, then that means unmount has been successful.

Also if cleanupResource() also handles deleting the PVC and waiting for the PV to disappear, then that means it needs to have been detached and deleted too.